app/src/main/java/io/gnosis/safe/ui/safe/share/ShareSafeDialog.kt

```diff
@@ -111,7 +110,7 @@ class ShareSafeDialog : BaseViewBindingDialogFragment<DialogShareSafeBinding>()
             safeFields.visible(false)
         }
         Timber.e(error)
-        snackbar(requireView(), Snackbar.LENGTH_LONG, R.string.error_invalid_safe)
```

Comment:
Why did this compile? Is Snackbar.LENGTH_LONG a valid StringRes?

Reply:
its an int, so yes